release-notes.hbs.md

```diff
@@ -57,33 +57,48 @@ This release includes the following changes, listed by component and area.
+    - Note that the Delivery section content will be conditionally displayed depending on whether the targetted environment includes the Deliverable object (Delivery will be present on environments created using the Iterate and Build installation profiles)
+  - Added a `Healthy` column to the Supply Chain resources table.
+    - The column values are color coded to indicate the health of each resource at-a-glance. 
   - Added an Overview section to show workload name and type.
-  - Each section is now indented under its corresponding header.
-  - Emojis are printed to distinguish each section.
-  - A new column to show the resource stamped out by the supply chain was also added.
-  - Deliverable information is being surfaced whenever it's available.
-  - Pods status is now same as Kubectl so, for example, if there are init containers, when `workload get` is used, the `Init` status of these will be printed in the output.
-- Local source changes will be updated/uploaded only if there is an actual change to the code.
-- Maven artifact is also supported via flags. It can be set through complex params or the new flags. Check [workload apply maven source flags](./cli-plugins/apps/command-reference/commands-details/workload_create_update_apply.hbs.md#a-idapply-maven-artifacta---maven-artifact) for more info about their usage.
-- There are some environment variable that can be set as default values for apps plugin flags. These are:
-  * `--type`: TANZU_APPS_TYPE
-  * `--registry-ca-cert`: TANZU_APPS_REGISTRY_CA_CERT
-  * `--registry-password`: TANZU_APPS_REGISTRY_PASSWORD
-  * `--registry-username`: TANZU_APPS_REGISTRY_USERNAME
-  * `--registry-token`: TANZU_APPS_REGISTRY_TOKEN
+  - Added Emojis to, and indentation under, each section header in the command output to better distinguish each section.
+  - Updated the STATUS column in the table within the Pods section so that it displays the `Init` status when there are init containers (instead of displaying a less helpful/accurate `pending` value).
+    - In fact, all column values in the Pods table have been updated so the output will be equivalent to the output from `kubectl get pod/pod-name`
+- Updated Go to its latest version (1.19)
 
 ##### <a id="apps-plugin-deprecations"> Deprecations
 
-- First warning that `workload update` command will be deprecated.
+- The `tanzu apps workload update` command will be deprecated in the `apps` CLI plugin. Please use `tanzu apps workload apply` instead.
+  - `update` will be deprecated in two TAP releases (in TAP v1.5.0) or in one year (on Oct 11, 2023), whichever is longer.
 
 #### <a id="src-cont-features"></a>Source Controller
 
-- Feature 1
-- Feature 2
+- Added support for `SNAPSHOT` versions in `MavenArtifacts`
+- Added support for  Maven Artifact version LATEST
+- Optimized `MavenArtifact` artifact download during interval sync.
```

Comment:
Replace this
```
Optimized `MavenArtifact` artifact download during interval sync.
```
with
```
'MavenArtifact' artifact download during interval sync has been optimized. Only after the SHA on the Maven Repository has changed will the source controller download the artifact. Otherwise, the download is skipped.
```